tool/dartpad_picker/compile_dev.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+: ${TMP:=tmp}
+
+set -x
+
+pub get
+pub run build_runner build --output $TMP --define "build_web_compilers|entrypoint=compiler=dart2js"
+cp $TMP/web/* ../../src/assets/dash/js
+rm -rf $TMP
```

Comment:
Just curious, when do you use this script (and why compile under dart2js and not the ddc)?

Reply:
Using DDC to debug was tricker than using dart2js in non-release mode. DDC modules expect package source code to be hosted at the root of the project (`localhost:4000/packages/blah`). I think I was seeing an issue where Jekyll wasn't serving the `src/packages/` directory correctly.